Lib/test/test_script_helper.py

```diff
@@ -81,20 +81,23 @@ def tearDown(self):
         # Reset the private cached state.
         script_helper.__dict__['__cached_interp_requires_environment'] = None
 
+    @mock.patch('os.environ', {})
```

Comment:
I would prefer to only modify add/remove variables needed by the function, instead of using a strange empty environment.

Reply:
Know what you mean, I'm on it.